<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Combinações</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1156</width>
        <height>749</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="SelectionWidget" name="layerSelection" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="SelectionWidget" name="fieldSelection" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="SelectionWidget" name="attributeSelection" native="true"/>
       </item>
       <item row="3" column="0">
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Camadas com Combinações</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="layersFoundTw">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>200</height>
               </size>
              </property>
              <column>
               <property name="text">
                <string>Caso</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Camada</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Quantidade de Feições</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SelectionWidget</class>
   <extends>QWidget</extends>
   <header>SAP_Operador.modules.combinationViewer.widgets.selectionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
